com.unity.render-pipelines.universal/Samples~/TeachingSamples/SharedAssets/Scripts/FPSController.cs

```diff
@@ -0,0 +1,150 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[RequireComponent(typeof(Rigidbody))]
+public class FPSController : MonoBehaviour
```

Comment:
I wasn't able to find any controller that required less maintenance and didn't add dependencies. If a controller that uses the new input system surfaces, It would probably be a good idea to adopt it and update the samples to the new system, but for now that is postponed.